cmd/kube-apiserver/app/encryption/encryption.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package encryptionconfig
+// Package encryption handles assigning encryption providers to each group resource storage.
+package encryption
```

Comment:
don't all apiservers depend on this encryption config? any resource can be encrypted (not just secrets, not just core kube-apiserver resources). moving this out of the `k8s.io/apiserver` component into the `cmd/kube-apiserver` package looks incorrect.